docs/content/doc/usage/agit-support.en-us.md

```diff
@@ -0,0 +1,58 @@
+---
+date: "	2022-09-01T20:50:42+0000"
+title: "Usage: Agit Setup"
+slug: "agit-setup"
+weight: 12
+toc: false
+draft: false
+menu:
+  sidebar:
+    parent: "usage"
+    name: "Agit Setup"
+    weight: 12
+    identifier: "agit-setup"
+---
+
+# Agit Setup
+
+In Gitea `1.13`, support for [agit](https://git-repo.info/en/2020/03/agit-flow-and-git-repo/) was added.
+
+## Creating PR with Agit
+
+Agit allows creating a PR while pushing code to the remote repo. This requires using a speacial command refspec.
+
+- `HEAD`
+
+  Target branch
+
+- `refs/<for|drafts|for-review>/`
+
+  Target PR Type.
+  - `for` - Normal PR
+  - `draft` - Draft PR
+  - `for-review` - Generate a PR ID for updating existing PR.
+
+- `<target-branch>/<session>`
+
+  Target remote branch to open a PR.
+
+- `-o <topic|title|description>`
+
+  Options for the PR
+  - `title` - Title of the PR.
+  - `topic` - Topic of the PR.
+  - `description` - Description of the PR. (Contents in Markdown format)
+
+## Examples
+
+Example of pushing a repo with a PR:
+
+```shell
+git push origin HEAD:refs/for/master
+```
+
+Example of pushing a repo with a PR `topic`, `title` and `description`:
+
+```shell
+git push origin HEAD:refs/for/master -o topic="Topic of my PR" -o title="Title of the PR" -o description="# The PR Description\nThis can be markdown formatted.\n[x] Ok"
+```
```

Comment:
````suggestion
Agit allows to create PRs while pushing code to the remote repo. \
This can be done by pushing to the branch followed by a specific refspec (a location identifier known to git). \
The following example illustrates this:

```shell
git push origin HEAD:refs/for/master
```

The command has the following structure:

- `HEAD`: The target branch
- `refs/<for|drafts|for-review>/<branch>`: the target PR type
  - `for` - Create a normal PR with `<branch>` as target branch
  - `draft` - Create a draft PR with `<branch>` as target branch
  - `for-review` - Update an existing PR
- `<branch>/<session>`: The target branch to open the PR
- `-o <topic|title|description>`: Options for the PR
  - `title` - The PR title
  - `topic` - The PR topic
  - `description` - The PR description (in Markdown format)

Here's another advanced example for creating a new PR targeting `master` with `topic`, `title`, and `description`:

```shell
git push origin HEAD:refs/for/master -o topic="Topic of my PR" -o title="Title of the PR" -o description="# The PR Description\nThis can be **any** markdown content.\n[x] Ok"
```
````
The only thing I'm not sure about is: Does `for-review` do what I think it does? I found both the previous description and the official docs pretty confusing…

Reply:
Well if i remember for gitea it acts the same as normal pull for now

Reply:
Ok Sir, I'll commit these changes.